AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Activity/Edit.cshtml

```diff
@@ -154,7 +154,7 @@
     <script type="text/javascript">
 
             System.config({
-                baseURL: location.protocol + "//" + location.host + '/ts/'
```

Comment:
I would tend to push this stuff down a level. So for instance on my projects I tend to have either a folder per controller with the associated files for views off that controller
![image](https://cloud.githubusercontent.com/assets/153912/11212048/a70a0bde-8cf0-11e5-8514-4f3e0f76aac5.png)

or I'll inset another level so that I can have things like Controls, Pages, Infrastructure in the scripts directory

![image](https://cloud.githubusercontent.com/assets/153912/11212122/ff3560f6-8cf0-11e5-866d-7082966c0e2f.png)

Obviously each file is namespaced in accordance with the hierarchy. Arranging javascript should be just like arranging "real" c# code.